src/odemis/gui/cont/streams.py

```diff
@@ -331,6 +335,20 @@ def _add_stream_setting_controls(self):
             conf = stream_config.get(vaname)
             self.add_setting_entry(vaname, va, hw_comp=None, conf=conf)
 
+    def _add_axis_controls(self):
+        """
+        Add controls for the axes that are connected to the stream
+        """
+        if hasattr(self.stream, "axis_map") and self.stream.axis_map:
+            # Add Axes (in same order as config)
+            axes_names = util.sorted_according_to(list(self.stream.axis_map.keys()), list(self._stream_config.keys()))
```

Comment:
This whole function should only use `axis_vas`, not `axis_map` and `_axis_vas`, which should both be private.

Reply:
Doesn't it need the real names from axis_map?

Reply:
Woops - nvm.